test/common/http/conn_manager_impl_test.cc

```diff
@@ -2397,6 +2401,74 @@ TEST_F(HttpConnectionManagerImplTest, RequestTimeoutIsDisarmedOnConnectionTermin
   EXPECT_EQ(0U, stats_.named_.downstream_rq_timeout_.value());
 }
 
+TEST_F(HttpConnectionManagerImplTest, NotInvokeStreamDeletionConfiguredZero) {
```

Comment:
nit: it's hard to tell from these test names what this test actually is testing for. Can you make the test name include the param name we are testing and/or add a `//` name above each test to clarify?